docs/get-invited-and-onboard-to-techpass.md

```diff
@@ -1,139 +1,100 @@
-# Get invited and onboard to TechPass
+# Get invited and onboard to TechPass (vendors and offshore users)
 
-This article guides you how to request for a TechPass account and onboard to it. 
+This guide explains how vendors and offshore users can request a TechPass account and complete onboarding.
 
-?>- Alternatively, public officers with a non-SE GSIB device can access the [**TechBiz Portal**](https://portal.techbiz.suite.gov.sg) to request for TechPass and SEED provisioning, which is optional. <br>- For more information, see [**TechBiz documentation**](https://docs.developer.tech.gov.sg/docs/techbiz-documentation/).
+?> For offshore users, SEED provisioning is **mandatory** unless you belong to an offshore development centre (ODC).
 
+---
 
-## Audience
+## Audience  
+Vendors and offshore users who require a TechPass account.
 
-Users who require a TechPass account but do not have a WOG account or a non-SE GSIB device.
+---
 
-## Prerequisites
+## Prerequisites  
 
-You need the following to get invited to TechPass and complete the onboarding:
+Ensure you have:  
+- A **standard organisational email address** (not LiteMail).  
+- Received the **TechPass onboarding email** (valid for 30 days).  
 
-- Check the flowchart on the [Onboard to TechPass](onboard-to-techpass) page to see if you're eligible to sign up via the Portal. If you can't sign up for TechPass account via TechPass Portal, you need to request to get invited.
-- Your organisational email address which has a standard mailbox and not a LiteMail.  
-- Before you onboard, ensure you have received the TechPass onboarding email and is still valid.
+?> Failure to register within **5 days** invalidates the link, and you must request a new invitation.
 
-?>- TechPass does not support email accounts which does not have an inbox. For example, LiteMail accounts. If you use such an email account, upgrade it to a standard mailbox before requesting for TechPass.<br>- If you do not see the TechPass onboarding email in your inbox, please check your Junk Email, Deleted Items or Archive folder.<br>- The onboarding email is valid for 30 days. If you do not onboard to TechPass within this 30 days, we will terminate your TechPass account, and you need to sign up again.
+---
 
+## Step 1: Request a TechPass account  
 
-## Step 1: Request for TechPass account
+1. Contact your **project manager** or **reporting officer** to request TechPass and SEED provisioning.  
+2. Provide the following details:  
+   - Organisational email address  
+   - Mobile phone number  
+   - Project name  
 
-1. Contact your project manager or the reporting officer to request for the provisioning of TechPass and SEED, which is optional.
```

Comment:
Mobile phone and project name is not needed. 
Actually I think can retain this `Contact your project manager or the reporting officer to request for the provisioning of TechPass and SEED, which is optional.` (seed is optional for onshore users, mandatory for offshore)